bin/runners

```
@@ -32,4 +32,9 @@ trap('SIGUSR2') do
   Bugsnag.notify(RunnersTimeoutError.new)
 end
 
-Runners::CLI.new(argv: ARGV.dup, stdout: STDOUT, stderr: STDERR).run
+begin
+  Runners::CLI.new(argv: ARGV.dup, stdout: STDOUT, stderr: STDERR).run
+rescue => exn
```

My understanding is that this change tries to handle `SystemExit`, isn't it? If so, I think `rescue` block should explicitly `SystemExit` because `rescue` block handles `StandardError` by default. Also, should we change `ignore_classes` to exclude `SystemExit` from it?

```suggestion
rescue StandardError, SystemExit => exn
```

---

> My understanding is that this change tries to handle `SystemExit`, isn't it?

No. This PR aims to report `StandardError`. Unless this added `rescue`, any `StandardError` errors are not reported. 😢

---

OK, I'm not sure. Let's do it :muscle: